resource/aws_fsx_windows_file_system: Increase default creation timeout

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform CLI and Terraform AWS Provider Version

```
Terraform CLI v0.12.29

Terraform AWS Provider main branch
```

### Affected Resource(s)

* aws_fsx_windows_file_system

### Terraform Configuration Files

```hcl
resource "aws_fsx_windows_file_system" "test" {
  active_directory_id = aws_directory_service_directory.test.id
  skip_final_backup   = true
  storage_capacity    = 32
  subnet_ids          = [aws_subnet.test1.id]
  throughput_capacity = 8
}
```

### Debug Output

Can provide if needed.

### Expected Behavior

Resource creation happens without timeout error for majority of cases.

### Actual Behavior

Very frequently hitting the default creation timeout of 30 minutes:

```
=== CONT  TestAccAWSDataSyncLocationFsxWindows_basic
TestAccAWSDataSyncLocationFsxWindows_basic: resource_aws_datasync_location_fsx_windows_file_system_test.go:86: Step 1/2 error: Error running apply: 2020/11/19 09:57:15 [DEBUG] Using modified User-Agent: Terraform/0.12.29 HashiCorp-terraform-exec/0.10.0
Error: Error waiting for filesystem (fs-00b5e00e73764e811) to become available: timeout while waiting for state to become 'AVAILABLE' (last state: 'CREATING', timeout: 30m0s)
--- FAIL: TestAccAWSDataSyncLocationFsxWindows_basic (4748.98s)

=== CONT  TestAccAWSDataSyncLocationFsxWindows_subdirectory
TestAccAWSDataSyncLocationFsxWindows_subdirectory: resource_aws_datasync_location_fsx_windows_file_system_test.go:147: Step 1/2 error: Error running apply:
Error: Error waiting for filesystem (fs-090dcf07d70912a72) to become available: timeout while waiting for state to become 'AVAILABLE' (last state: 'CREATING', timeout: 30m0s)
--- FAIL: TestAccAWSDataSyncLocationFsxWindows_subdirectory (4186.00s)

=== CONT  TestAccAWSFsxWindowsFileSystem_StorageCapacity
TestAccAWSFsxWindowsFileSystem_StorageCapacity: resource_aws_fsx_windows_file_system_test.go:535: Step 1/3 error: Error running apply:
Error: Error waiting for filesystem (fs-0afc18f052141e201) to become available: timeout while waiting for state to become 'AVAILABLE' (last state: 'CREATING', timeout: 30m0s)
--- FAIL: TestAccAWSFsxWindowsFileSystem_StorageCapacity (4006.30s)
```

Recommend bumping it to 45 or 60 minutes in the resource code and updating the documentation.

### Steps to Reproduce

1. `TF_ACC=1 go test -run='(TestAccAWSFsxWindowsFileSystem_|TestAccAWSDataSyncLocationFsxWindows_)' -parallel=20 -timeout=4h -v`

### Important Factoids

Could only be an issue when there are lots of parallel creations.

### References

* [Contributing Guide on Running/Writing Acceptance Testing](https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/contributing/running-and-writing-acceptance-tests.md)

--- a/aws/resource_aws_fsx_windows_file_system.go
+++ b/aws/resource_aws_fsx_windows_file_system.go
@@ -29,7 +29,7 @@ func resourceAwsFsxWindowsFileSystem() *schema.Resource {
 		},
 
 		Timeouts: &schema.ResourceTimeout{
-			Create: schema.DefaultTimeout(30 * time.Minute),
+			Create: schema.DefaultTimeout(45 * time.Minute),
 			Delete: schema.DefaultTimeout(30 * time.Minute),
 		},
 

--- a/website/docs/r/fsx_windows_file_system.html.markdown
+++ b/website/docs/r/fsx_windows_file_system.html.markdown
@@ -98,7 +98,7 @@ In addition to all arguments above, the following attributes are exported:
 `aws_fsx_windows_file_system` provides the following [Timeouts](/docs/configuration/resources.html#timeouts)
 configuration options:
 
-* `create` - (Default `30m`) How long to wait for the file system to be created.
+* `create` - (Default `45m`) How long to wait for the file system to be created.
 * `delete` - (Default `30m`) How long to wait for the file system to be deleted.
 
 ## Import

